container/container.go

```diff
@@ -227,3 +234,51 @@ func (a *AvroContainerWriter) AddAvroContainerWriter(p *generator.Package) {
 	p.AddFunction(a.filename(), a.name(), "WriteRecord", a.writeRecordDef())
 	p.AddFunction(a.filename(), a.name(), "Flush", a.flushDef())
 }
+
+func (a *AvroContainerWriter) AddCheckSchema(p *generator.Package) {
```

Comment:
Yeah, you're actually right. This should actually just be run as a one time thing because ultimately the reason we would want to verify is because we pushed a change to the client code repo, updated the consumer and redeployed it. We can expect the state to always be the same in between deploys.